code/modules/mob/living/basic/space_fauna/lightgeist.dm

```diff
@@ -57,19 +65,15 @@
 		complete_text = "%TARGET%'s wounds mend together.",\
 	)
 
-	remove_verb(src, /mob/living/verb/pulled)
-	remove_verb(src, /mob/verb/me_verb)
-	var/datum/atom_hud/medsensor = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
-	medsensor.show_to(src)
-
-	ADD_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
 
-/mob/living/simple_animal/hostile/lightgeist/ghost()
+/mob/living/basic/lightgeist/ghost()
 	. = ..()
 	if(.)
 		death()
 
-/mob/living/simple_animal/hostile/lightgeist/AttackingTarget()
-	if(istype(target, /obj/structure/ladder)) //special case where lightgeists can use ladders properly.
-		var/obj/structure/ladder/laddy = target
-		laddy.use(src)
+/// This is a bit neutered since these aren't intended to exist outside of player control, but it's a bit weird to just have these guys be completely stationary.
+/// No attacking or anything like that, though. Just something so they seem alive.
```

Comment:
They're used in the clog event tho, right? don't they need ai behavior?

Reply:
![image](https://github.com/tgstation/tgstation/assets/34697715/d2ed3366-7031-4848-b667-f7f07a819c79)


the old implementation had them do absolutely nothing except just stand there. this is an improvement